### PR TITLE
can use pl33 filter with dask-backed DataArrays

### PR DESCRIPTION
**Pull request by kthyng**

### CODE REVIEW

File: oceans/filters.py [hunk 1]
```diff
@@ -603,6 +604,16 @@ def pl33tn(x, dt=1.0, T=33.0, mode="valid"):
...
 
+    import cf_xarray  # noqa: F401
+    import xarray as xr
+
+    if isinstance(x, xr.Dataset):
+        raise TypeError("Input a DataArray not a Dataset.")
+
+    if isinstance(x, xr.DataArray):
+        # find dt in units of hours
+        dt = (x.cf["T"][1] - x.cf["T"][0]) * 1e-9 / 3600
```
Comment (ocefpaf) [inline oceans/filters.py:615]: This will get clobbered below, no?

Comment (kthyng) [reply oceans/filters.py:615]: Those other lines use an input `dt` though, they just assume 1 hour, then modify the dt according to T.

Comment (kthyng) [reply oceans/filters.py:615]: (though I did test with an hourly dataset so...)

**Comment by ocefpaf** [comment]
> Ok, but can I ping you to review still?

Sure. I owe some other reviews but this one will be quicker :grimacing:

**Comment by kthyng** [comment]
@ocefpaf Should I merge?

**Comment by ocefpaf** [comment]
I'll try to get a new release out soon. Probably next Tuesday.

**Pull request merged**